ci/cpu/prebuild.sh

```diff
@@ -0,0 +1 @@
+#!/usr/bin/env bash
```

Comment:
Does this package need to be uploaded for all CUDA/Python combos?

Reply:
It's python only, so yes.